src/plugins/home/server/services/sample_data/sample_data_installer.ts

```diff
@@ -155,13 +155,17 @@ export class SampleDataInstaller {
 
   private async installDataIndex(dataset: SampleDatasetSchema, dataIndex: DataIndexSchema) {
     const index = createIndexName(dataset.id, dataIndex.id);
+    const statefulOffering = dataset.settingsForIndex
```

Comment:
This var contains index settings, so I think calling it `indexSettings` would make more sense 😊  
From my above comment (naming `withIndexSettings`) this could then be

```ts
const indexSettings = dataset.withIndexSettings
  ? { number_of_shards: 1, auto_expand_replicas: '0-1' }
  : {};
```